Britannia/common/landed_titles/00_landed_titles.txt

```diff
@@ -22195,20 +22195,31 @@ e_britannia = {
 				name_list_romanoegyptian = cn_brigantia_greek
 				name_list_oldirish = cn_ebrauc_gaelic
 				name_list_gaelic = cn_ebrauc_gaelic
+				name_list_pictish = cn_ebrauc_pictish
 			}
 			
 			c_eburacum = {
 				color={ 80 115 65 }
 			
 				cultural_names = {
-					name_list_greek = cn_eburacum_greek
+					name_list_roman = cn_eburacum
```

Comment:
Missing Romano-British in the Latin localisations I think?

Reply:
Fixed